website/src/views/NotFound/NotFound.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { NotFound } from '..';
+import '../../utils/tests/test.css';
+
+describe('NotFound View', () => {
+  it('Expect the NotFound page to match snap shot', () => { // eslint-disable-line no-undef
+    const component = renderer.create(
+      <NotFound />,
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot(); // eslint-disable-line no-undef
```

Comment:
These lines are unnecessary, since they can be completed as one line of code.

Reply:
yeah but `expect(component.toJSON()).toMatchSnapshot();` seems like it has to much going on.